wgsl/index.bs

```diff
@@ -11909,6 +11909,7 @@ the rules in [[#floating-point-overflow]] apply.
 The accuracy of an [=AbstractFloat=] operation is as follows:
 * A correct result is required when the corresponding [=f32=] operation requires a correct result.
 * A [=correctly rounded=] result is required when the corresponding [=f32=] operation requires a correctly rounded result.
+* `fract(x)`'s error is inherited from `x - floor(x)`, where the intermediate calculations are performed as [=AbstractFloat=] operations.
```

Comment:
MSL says `fract` is correctly rounded.
Vulkan doesn't have a GLSLStd450 extended instruction for fract.  

I wondered if we can simplify this and say fract is correctly rounded.

Then @dj2 reminded me that if WGSL ever gets `f64` then AbstractFloat should be compatible with what `f64` would be on the GPU.   So it's better to do it the way you have proposed it. That is, a GPU implementation of `fract` could be inherited implementation (with f64 operations and intermediates).